node-src/git/getChangedFilesWithReplacement.ts

```diff
@@ -39,7 +39,8 @@ export async function getChangedFilesWithReplacement(
       `Got error fetching commit for #${build.number}(${build.commit}): ${err.message}`
     );
 
-    if (/(bad object|uncommitted changes)/.test(err.message)) {
+    console.error(err.message);
+    if (/(bad object|uncommitted changes|timeout)/.test(err.message)) {
```

Comment:
We'd also need to trace the propagation, but it looks like `execa` sets `error.timedOut` on a timeout, so we could check that instead of string matching for `timeout`

https://github.com/sindresorhus/execa/blob/main/docs/termination.md#timeout

Reply:
@jmhobbs, do you have recommendations on how I would do that?
Would it be something similar to this?
`if (/(bad object|uncommitted changes)/.test(err.message) || err.timedOut) {`

Reply:
What is the test for `bad object` and `uncommitted changes` looking for?  Have we seen issues with other errors from `git` as well as the time out?

Reply:
@jmhobbs The `bad object` is quite common when we can't find a commit and is the basis for our replacement build logic. The `uncommitted changes` are for the VTA.

I've never seen this timeout issue before.